examples/step-14/step-14.cc

```diff
@@ -2576,6 +2576,11 @@ namespace Step14
                           WeightedResidualCopyData                          &copy_data,
                           FaceIntegrals                                     &face_integrals) const
     {
+      // Because of WorkStream, estimate_on_one_cell requires a CopyData object
+      // even if it is no used. The next line silence a warning about this unused
+      // variable.
+      (void) copy_data;
```

Comment:
silence -> silences

We could also just not name the second to last argument in the function definition.